app/src/main/res/layout/onboarding_manual_signin.xml

```diff
@@ -33,14 +33,14 @@
         app:layout_constraintStart_toStartOf="@id/avatar_icon"
         app:layout_constraintBottom_toBottomOf="@id/avatar_icon"
         android:layout_marginStart="52dp"
-        tools:text="@string/onboarding_account_sign_in_header_1" />
+        tools:text="@string/onboarding_account_sign_in_header" />
```

Comment:
Can we set the `android:text` directly here? I imagine we used to have to set the text in the ViewHolder because we are filling a %s string parameter for a string such as "Welcome to %s!". %s needs to be refer to the app name which could be different for the different releases (Firefox Preview, Firefox Nightly, Firefox Beta, etc). Since the string is constant now, we should be able to just set it directly in the xml and remove the text being set in the viewholder.